src/clarity-angular/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -0,0 +1,292 @@
+            align-self: flex-end;
+
+            //Dims
+            border: none;
+            height: $vertical-nav-item-height;
+            padding: 0;
+
+            //Others
+            background-color: transparent;
+            cursor: pointer;
+            outline-offset: -5px;
+        }
+
+        .nav-trigger {
+            //Display
+            display: flex;
+            justify-content: center;
+            align-items: center;
+
+            //Dims
+            margin-top: -1 * $vertical-nav-top-padding;
+            height: $vertical-nav-item-height;
+            width: $vertical-nav-item-height;
+        }
+
+        .nav-group-trigger,
+        .nav-link {
+            //Display
+            display: flex;
+
+            //Dims
+            height: $vertical-nav-item-height;
+            padding: 0 $vertical-nav-link-padding-right 0 $vertical-nav-link-padding-left;
+
+            //Others
+            line-height: $vertical-nav-item-height;
+            outline-offset: -5px;
+
+            @include vertical-nav-link-colors();
+
+            &:hover .nav-icon {
+                fill: $clr-action-blue;
+            }
+        }
+
+        .nav-link {
+            flex: 0 0 $vertical-nav-item-height;
+        }
+
+        //Parent .nav-group
+        .nav-group-trigger {
+            //Dims
+            width: 100%;
+
+            //Others
+            text-align: left;
+        }
+
+        //Parent .nav-trigger
+        .nav-trigger-icon {
+            //Display
+            align-self: center;
+
+            //Others
+            transition: all 0.2s ease-in-out;
+        }
+
+        .nav-header {
+            //Dims
+            padding: 0 $vertical-nav-link-padding-right 0 $vertical-nav-link-padding-left;
+
+            //Others
+            @include clr-getTypePropertiesForDomElement(vertical_nav_header, (font-size, font-weight, letter-spacing));
+            line-height: $vertical-nav-item-height;
+        }
+
+        //Parent .nav-link
+        .nav-text {
+            //Display
+            flex: 1 1 auto;
+
+            //Mixins
+            @include overflow-ellipsis();
+        }
+
+        //Parent .nav-text
+        .nav-icon {
+            //Display
+            flex: 0 0 $vertical-nav-icon-size;
+            align-self: center;
+
+            //Dims
+            height: $vertical-nav-icon-size;
+            width: $vertical-nav-icon-size;
+            margin-right: $vertical-nav-icon-right-margin;
+
+            //Others
+            vertical-align: middle;
+        }
+
+        //Parent .nav-group-trigger
+        .clr-nav-caret {
+            $caret-left-padding: $vertical-nav-link-padding-right;
+
+            //Display
+            flex: 0 0 $vertical-nav-icon-size + $caret-left-padding;
+            align-self: center;
+
+            //Dims
+            margin-left: auto;
+            padding-left: $caret-left-padding;
+
+            //Others
+            vertical-align: middle;
+
+            clr-icon {
```

Comment:
it is but we want to add styles to the specific class within the nav and not clr-icon in general.